File: 229-majority-element-ii/majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        my_dict = {}
        size = floor(len(nums) / 3)
        ans = []

        for item in nums:
            if item not in my_dict:
                my_dict[item] = 1
            else:
                my_dict[item] +=1

        for key, value in my_dict.items():
            if value > size:
                ans.append(key)
            
        return ans

File: 229-majority-element-ii/majority-element-ii.ts
function majorityElement(nums: number[]): number[] {
    let my_dict: {[key: number]: number} = {}
    const size: number = Math.floor(nums.length / 3)
    let ans: number []  = []

    nums.forEach((item: number) => {
        if (my_dict.hasOwnProperty(item)) my_dict[item] += 1;
        else my_dict[item] = 1;
    });

    for (const [key, value] of Object.entries(my_dict)){
        if (value > size) ans.push(Number(key));
    };

    return ans;
};